[0.6]: test_roundtrip fails on ppc64el

Testing the verison 0.6, installes on PowerPC 64 bit little endian gives me the following failure:

```
________________________________ test_roundtrip ________________________________

tmpdir = local('/tmp/pytest-of-debci/pytest-0/test_roundtrip0')

    def test_roundtrip(tmpdir):
        filenames = get_pkg_data_filenames('data', pattern='*.reg')
    
        # AstropyUserWarning will be emitted only for some of the files
        with warnings.catch_warnings():
            warnings.simplefilter('ignore', AstropyUserWarning)
            # Check that all test files are readable
            for filename in filenames:
                regions = Regions.read(filename, format='ds9')
    
        tempfile = tmpdir.join('tmp.ds9').strpath
>       regions.write(tempfile, format='ds9', overwrite=True, precision=20)

/usr/lib/python3/dist-packages/regions/io/ds9/tests/test_ds9.py:35: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
/usr/lib/python3/dist-packages/regions/core/regions.py:234: in write
    return RegionsRegistry.write(self.regions, filename,
/usr/lib/python3/dist-packages/regions/core/registry.py:116: in write
    return writer(regions, filename, **kwargs)
/usr/lib/python3/dist-packages/regions/io/ds9/write.py:109: in _write_ds9
    output = _serialize_ds9(regions, precision=precision)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

regions = [], precision = 20

    @RegionsRegistry.register(Region, 'serialize', 'ds9')
    @RegionsRegistry.register(Regions, 'serialize', 'ds9')
    def _serialize_ds9(regions, precision=8):
        region_data = []
        for region in regions:
            if isinstance(region, (CompoundPixelRegion, CompoundSkyRegion)):
                warnings.warn('Cannot serialize a compound region, skipping',
                              AstropyUserWarning)
    
            if isinstance(region, RegularPolygonPixelRegion):
                region = region.to_polygon()
    
            region_data.append(_serialize_region_ds9(region, precision=precision))
    
        # ds9 file header
        output = '# Region file format: DS9 astropy/regions\n'
    
        # extract common region metadata and place in the global metadata
        all_meta = []
        for region in region_data:
            region_meta = deepcopy(region['meta'])
            region_meta.pop('tag', None)  # "tag" cannot be in global metadata
            all_meta.append(region_meta)
    
>       global_meta = dict(set.intersection(*[set(meta_dict.items())
                                              for meta_dict in all_meta]))
E       TypeError: unbound method set.intersection() needs an argument

/usr/lib/python3/dist-packages/regions/io/ds9/write.py:45: TypeError
```
While I am not sure why this happens only on this platform (and I could imagine  that this hides another, platform specific issue), I think that the mentioned line is really problematic https://github.com/astropy/regions/blob/8d3b9fab4c93041c918552a0dd89baac38a1d9eb/regions/io/ds9/write.py#L45-L48

Trying to call `_serialize_ds9` with an empty region list will lead to this error.

Python version is 3.9. Full test log [here](https://ci.debian.net/data/autopkgtest/testing/ppc64el/a/astropy-regions/20367633/log.gz).

--- a/CHANGES.rst
+++ b/CHANGES.rst
@@ -12,6 +12,9 @@ Bug Fixes
 
 - Fixed an issue with the CRTF for certain malformed input. [#448]
 
+- Fixed a bug where DS9 serialization of an empty list raised an error.
+  [#449]
+
 API Changes
 -----------
 

--- a/regions/io/ds9/tests/test_ds9.py
+++ b/regions/io/ds9/tests/test_ds9.py
@@ -595,3 +595,8 @@ def test_serialize_regularpolygon():
     result1 = region.serialize(format='ds9')
     result2 = poly_region.serialize(format='ds9')
     assert result1 == result2
+
+
+def test_serialize_empty_list():
+    regions = Regions([])
+    assert regions.serialize(format='ds9') == ''

--- a/regions/io/ds9/write.py
+++ b/regions/io/ds9/write.py
@@ -21,6 +21,9 @@ __all__ = []
 @RegionsRegistry.register(Region, 'serialize', 'ds9')
 @RegionsRegistry.register(Regions, 'serialize', 'ds9')
 def _serialize_ds9(regions, precision=8):
+    if not regions:
+        return ''
+
     region_data = []
     for region in regions:
         if isinstance(region, (CompoundPixelRegion, CompoundSkyRegion)):

